net/tayga/src/opnsense/mvc/app/models/OPNsense/Tayga/General.xml

```diff
@@ -1,27 +1,31 @@
 <model>
     <mount>//OPNsense/tayga/general</mount>
     <description>Tayga configuration</description>
-    <version>0.0.2</version>
+    <version>0.0.3</version>
     <items>
         <enabled type="BooleanField">
             <default>0</default>
             <Required>Y</Required>
         </enabled>
         <v4address type="NetworkField">
             <default>192.168.255.1</default>
-            <Required>N</Required>
+            <Required>Y</Required>
         </v4address>
+        <v4destination type="NetworkField">
+            <default>192.168.254.1</default>
+            <Required>Y</Required>
+        </v4destination>
         <v6address type="NetworkField">
             <default>2001:db8:1::2</default>
```

Comment:
There is also a private range in V6 which may be better to use: fc00/7. @fichtner, @AdSchellevis?

https://tools.ietf.org/html/rfc4193#page-3

Reply:
@fabianfrz 2001:db8 seems to be used as default in Tayga's example config, other vendors (Cisco, Palo Alto, ..) seem to be using the same prefix in the same type of examples as well.

Reply:
I changed it to well-known